### PR TITLE
Fix two typos in picture archivist in f

### PR DESCRIPTION
**Pull request by bender2k14**

### CODE REVIEW

File: _posts/2019-09-16-picture-archivist-in-f.html [hunk 1]
```diff
@@ -306,7 +306,7 @@ <h3 id="abe95ba6865745bc9df8004079d8a250">
 		Calculating moves <a href="#abe95ba6865745bc9df8004079d8a250" title="permalink">#</a>
 	</h3>
 	<p>
-		One pure step remains. The result of calling the <code>moveTo</code> function is a tree with the desired structure. In order to actually move the files, though, for each file you'll need to keep track of both the source path and the destination path. To make that explicit, you can define a type for that purpose:
+		One pure step remains. The result of calling the <code>moveTo</code> function given a tree with the desired structure. In order to actually move the files, though, for each file you'll need to keep track of both the source path and the destination path. To make that explicit, you can define a type for that purpose:
```
Comment (ploeh) [inline _posts/2019-09-16-picture-archivist-in-f.html:309]: This isn't a typo 😄

Comment (bender2k14) [reply _posts/2019-09-16-picture-archivist-in-f.html]: Oh, sure.  I see it now.  Thanks for pointing this out.  I removed the commit from this branch that made this change.  There is still the other commit (that I think fixes a typo ;) ).

File: _posts/2019-09-16-picture-archivist-in-f.html [hunk 2]
```diff
@@ -484,7 +484,7 @@ <h3 id="f30093164b184bbf877f307fa4cf4c63">
 		First, you load the <code>sourceTree</code> using the <code>readTree</code> operation. This returns a <code>Tree&lt;string,string&gt;</code>, so map the leaves to <code>FileInfo</code> objects. You then load the image metatadata by traversing <code>sourceTree</code> with <code>Tree.choose readPhoto</code>. Each call to <code>readPhoto</code> produces a <code>PhotoFile option</code>, so this is where you want to use <code>Tree.choose</code> to throw all the <code>None</code> values away.
 	</p>
 	<p>
-		Those two lines of code is the initial impure step of the sandwich (yes: mixed metaphors, I know).
+		Those two lines of code are the initial impure step of the sandwich (yes: mixed metaphors, I know).
```
Comment (ploeh) [inline _posts/2019-09-16-picture-archivist-in-f.html:487]: This is a problematic sentence, I admit. There are two lines of code (plural), but one step (singular). How about "Those two lines of code constitute the initial impure step"?

Comment (bender2k14) [reply _posts/2019-09-16-picture-archivist-in-f.html]: Yes, that works.  I just force pushed that change.

**Pull request merged**